src/amo/components/AddonReviewCard/index.js

```diff
@@ -71,26 +72,20 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
   static defaultProps = {
     _config: config,
     flaggable: true,
+    smallerWriteReviewButton: false,
```

Comment:
I guess it's not worth changing but making them `puffy` seems more like an exception rather than the norm.

Reply:
Yeah, that makes sense. It's a small change so I've fixed that.